### PR TITLE
Observability Dashboard - Overview right sidebar

### PR DESCRIPTION
**Pull request by Minoli**
## Purpose
-Add overview sidebar
-Fix datatables responsive issue

### CODE REVIEW

File: system/control-plane/global/components/observability/org.wso2.vick.observability.ui/src/pages/overview/SidePanelContent.js [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+import TableCell from "@material-ui/core/TableCell";
+import TableHead from "@material-ui/core/TableHead";
+import TableRow from "@material-ui/core/TableRow";
+import Timeline from "@material-ui/icons/Timeline";
+import Typography from "@material-ui/core/Typography";
+import withGlobalState from "../common/state";
+import {withStyles} from "@material-ui/core/styles";
+import withColor, {ColorGenerator} from "../common/color";
+
+
```
Comment (jeradrutnam) [inline system/control-plane/global/components/observability/org.wso2.vick.observability.ui/src/pages/overview/SidePanelContent.js]: Remove additional newline.

File: system/control-plane/global/components/observability/org.wso2.vick.observability.ui/src/pages/overview/index.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    display: none;
+}
+
+.overviewSidebarListTable tfoot td:last-child {
+    padding-right: 0;
+}
+
+.overviewSidebarListTable td {
+    padding: 0
+}
```
Comment (jeradrutnam) [inline system/control-plane/global/components/observability/org.wso2.vick.observability.ui/src/pages/overview/index.css]: Missing newline end.

File: system/control-plane/global/components/observability/org.wso2.vick.observability.ui/src/pages/overview/SidePanelContent.js (hunk 1 above)
Comment (Minoli) [reply system/control-plane/global/components/observability/org.wso2.vick.observability.ui/src/pages/overview/SidePanelContent.js]: fixed

File: system/control-plane/global/components/observability/org.wso2.vick.observability.ui/src/pages/overview/SidePanelContent.js [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+    secondaryHeading: {
+        paddingRight: 10
+    },
+    panelDetails: {
+        padding: 0,
+        marginBottom: 100
+    },
+    sidebarListTableText: {
+        fontSize: 12
+
```
Comment (dakshika) [inline system/control-plane/global/components/observability/org.wso2.vick.observability.ui/src/pages/overview/SidePanelContent.js]: unwanted blank line

File: system/control-plane/global/components/observability/org.wso2.vick.observability.ui/src/pages/overview/SidePanelContent.js [hunk 4]
```diff
@@ -0,0 +1,292 @@
...
+                        className={classes.expansionSum}>
+                        {summary.content[1].value === 0
+                            ? ""
+                            : <Typography className={classes.secondaryHeading}><CheckCircleOutline
+                                className={classes.cellIcon}
+                                style={{
+                                    color: colorGenerator.getColor(ColorGenerator.SUCCESS)
+                                }}/> {summary.content[1].value}
+                            </Typography>}
+                        {summary.content[2].value === 0
```
Comment (dakshika) [inline system/control-plane/global/components/observability/org.wso2.vick.observability.ui/src/pages/overview/SidePanelContent.js]: better to get this as a variable

File: system/control-plane/global/components/observability/org.wso2.vick.observability.ui/src/pages/overview/SidePanelContent.js (hunk 3 above)
Comment (Minoli) [reply system/control-plane/global/components/observability/org.wso2.vick.observability.ui/src/pages/overview/SidePanelContent.js]: Removed in development repo https://github.com/wso2-cellery/mesh-observability/blob/master/components/global/portal/io.cellery.observability.ui/src/components/overview/SidePanelContent.js

**Pull request merged**